website/docs/using-unleash/compliance/fedramp.mdx

```diff
@@ -0,0 +1,54 @@
+| [AC-07 Unsuccessful Logon Attempts](https://csrc.nist.gov/projects/cprt/catalog#/cprt/framework/version/SP_800_53_5_1_0/home?element=AC-7) | Unleash restricts user logins after 10 failed attempts.                                                                                                                                                                                                                                                                      |
+
+## Audit and Accountability
+
+| **FedRAMP Control**                                                                                      | **Unleash Features**                                                                                                                                                                                |
+|----------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [AU-02 Event Logging](https://csrc.nist.gov/projects/cprt/catalog#/cprt/framework/version/SP_800_53_5_1_0/home?element=AU-2)           | Unleash provides detailed [audit logs and event tracking](/reference/events), accessible through the Admin UI or exportable for integration with other systems.                                      |
+| [AU-12 Audit Record Generation](https://csrc.nist.gov/projects/cprt/catalog#/cprt/framework/version/SP_800_53_5_1_0/home?element=AU-12) | Unleash provides detailed [audit logs and event tracking](/reference/events) to support auditors with the required evidence. Depending on the audit automation level, the evidence can be collected using the Unleash Admin UI, or exported to other systems. |
+
+## Security Assessment and Authorization
+
+| **FedRAMP Control**                                                                             | **Unleash Features**                                                                                                                       |
+|-------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [CA-8 Penetration Testing](https://csrc.nist.gov/projects/cprt/catalog#/cprt/framework/version/SP_800_53_5_1_0/home?element=CA-8) | Unleash conducts annual penetration testing by external auditors. Results are available for download from the Trust Center.                                                     |
```

Comment:
@dsusa72 should add any links here for trust center or just leave it as is?

Reply:
I think the Trust Center link would add value if the readers know what kind of information we share there and the access restrictions (you need to have mNDA with Unleash, i.e. you're in the sales process). Most readers would not know this. So I would just mention: Please contact Sales Engineering to get access to Trust Center (SE is a specific role and sales team will immediately understand and handle appropriately)

Reply:
I would just say "Available upon request" and link that to the sales contact us form. The Sales engineering piece is too specific for this doc, imo.